### DISCUSSION TITLE
Buckets and tables are empty after restoring project

### DISCUSSION THREAD

**Discussion by leo-fe99** [supabase/supabase#9033] [Questions] [answered]
The title is quite explanatory.
The project has been paused and I restored it, it took around `15/20 min`. 
Now I'm allowed to navigate through the dashboard, but buckets and tables (public schema) are not available.
**Link to the project:** https://app.supabase.com/project/jtzlelzqvysufttnlslv

**Comment by GaryAustin1** [comment, marked as answer]
You should contact support if you have not already.

**Reply by leo-fe99** [reply]
@GaryAustin1 
Done on friday, no answer yet...

**Reply by leo-fe99** [reply]
They resolved the problem. 
Still don't know what happened.
